docs/user-guide.md

```diff
@@ -0,0 +1,223 @@
+* The `openshift-install` version mentioned above does not include the config-image API. This API is available starting from version `4.14`.
+* * Download and extract the latest `openshift-install` version from: https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/latest/openshift-install-linux.tar.gz
+```shell
+  curl -O https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/latest/openshift-install-linux.tar.gz
+  tar -xvf openshift-install-linux.tar.gz
+  ```
+
+### Generate a Cluster Configuration Image
+* Create a configuration directory
+
+**:warning: use absolute paths.**
+  ```shell
+  export CLUSTER_CONFIG=/home/test/cluster_config
+  mkdir $CLUSTER_CONFIG && cd $CLUSTER_CONFIG
+  ```
+* Place both `install-config.yaml` and `agent-config.yaml` files in that directory.
+* Find examples in:
+  * [Appliance README](../README.md#examples)
+  * [OpenShift Documentation](https://docs.openshift.com/container-platform/4.13/installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#installation-bare-metal-agent-installer-config-yaml_preparing-to-install-with-agent-based-installer)
+  * [Static Networking](https://docs.openshift.com/container-platform/4.13/installing/installing_with_agent_ba[…]taller/preparing-to-install-with-agent-based-installer.html), currently blocked by [OCPBUGS-15637](https://issues.redhat.com/browse/OCPBUGS-15637)
+* When ready, generate the config-iso.
+  ```shell
+  ./openshift-install agent create config-image --dir $CLUSTER_CONFIG
+  ```
+* Note: The config ISO contains configurations and cannot be used as a bootable ISO.
+
+The content of `cluster_config` directory should be
+  ```shell
+  ├── agentconfig.noarch.iso
+  ├── auth
+  │   ├── kubeadmin-password
+  │   └── kubeconfig
+  ```
+
+### Mount
+* Mount the `agentconfig.noarch.iso` as a CD-ROM on every node, or attach it using a USB stick.
+* Start the machine(s)
+
+### Demo
+[![asciicast](https://asciinema.org/a/590824.svg)](https://asciinema.org/a/590824)
+
+### Monitor installation
+#### Use `openshift-install` to monitor the bootstrap process
+  ```shell
+  export KUBECONFIG=$CLUSTER_CONFIG/auth/kubeconfig 
+  ./openshift-install --dir $CLUSTER_CONFIG  wait-for bootstrap-complete
+  ```
+* Review OpenShift documentation: [Waiting for the bootstrap process to complete](https://docs.openshift.com/container-platform/4.13/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-installing-bare-metal_installing-platform-agnostic)
+#### Monitor the installation process
+  ```shell
+  ./openshift-install --dir $CLUSTER_CONFIG wait-for install-complete
+  ```
+* Review OpenShift documentation: [Completing installation on user-provisioned infrastructure](https://docs.openshift.com/container-platform/4.13/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-complete-user-infra_installing-platform-agnostic)
+
+### Recovery / Reinstall
```

Comment:
redundant section

Reply:
redundant with what? it's here to illustrate what the grub menu looks like.

Reply:
Yeah it just to explicitly explain about the functionality. As it's meant for the end user (as opposed to the README which is more dev oriented)

Reply:
We won't completely remove it.. but still considering renaming:)